pkgs/applications/kde/kamoso.nix

```diff
@@ -0,0 +1,40 @@
+{ mkDerivation
+, lib
+, extra-cmake-modules
+, kdoctools
+, wrapQtAppsHook
+, qtdeclarative
+, qtgraphicaleffects
+, qtquickcontrols2
+, kirigami2
+, kpurpose
+, gst_all_1
+, pcre
+}:
+
+let
+  gst = with gst_all_1; [ gstreamer gst-libav gst-plugins-base gst-plugins-good gst-plugins-bad ];
+
+in
+mkDerivation {
```

Comment:
```suggestion
mkDerivation rec {
```

Reply:
We need `gst` for the wrapper, so this won't fly.